SPECS/e2fsprogs/e2fsprogs.spec

```diff
@@ -62,6 +62,9 @@ rm -rf %{buildroot}%{_infodir}
 %find_lang %{name}
 
 %check
+# This test is known to fail; remove it
```

Comment:
Could you also add a link to the LFS page and the [upstream issue](https://github.com/tytso/e2fsprogs/issues/134) opened for it to this comment? This way, it's going to be easier to tell in the future, if we can re-enable the test.

Reply:
Sure; updated